test/units/util.sh

```diff
@@ -379,6 +380,14 @@ EOF
         chmod +x "$initdir/opt/script1.sh"
         echo MARKER=1 >"$initdir/usr/lib/systemd/system/other_file"
         mksquashfs "$initdir" /tmp/app1.raw -noappend
+        veritysetup format /tmp/app1.raw /tmp/app1.verity --root-hash-file /tmp/app1.roothash
```

Comment:
I had kept this one off verity intentionally, to ensure more code paths are tested. Can you add a third one instead?